biosimulations/apps/dispatch-service/src/app/app.controller.ts

```diff
@@ -154,13 +156,33 @@ export class AppController {
     this.jobMonitorCronJob(slurmjobId, simId, 10);
   }
 
+
   async jobMonitorCronJob(jobId: string, simId: string, seconds: number) {
     const job = new CronJob(`${seconds.toString()} * * * * *`, async () => {
       const jobStatus: DispatchSimulationStatus =
         (await this.simulationService.getSimulationStatus(jobId)) ||
         DispatchSimulationStatus.QUEUED;
       this.logger.log(`SLURM status for job ${jobId}: ${jobStatus}`);
-      this.modelsService.updateStatus(simId, jobStatus);
+
+      // Send the updated status to dispatchAPI
+      this.http.post(`${process.env.AUTH0_DOMAIN}oauth/token`, { "client_id": process.env.CLIENT_ID, "client_secret": process.env.CLIENT_SECRET, "audience": process.env.API_AUDIENCE, "grant_type": "client_credentials" }).subscribe((data: any) => {
```

Comment:
Could you grab these from the dispatch service rather than  proccess.env?

Reply:
Sure, I will push shortly with the required changes.